src/supplemental/quic/CMakeLists.txt

```diff
@@ -35,8 +35,9 @@ if (NNG_ENABLE_QUIC)
 
     nng_include_directories(${CMAKE_SOURCE_DIR}/build/_deps/opensslquic-build/openssl/include)
     ## link openssl through target
-    # nng_link_libraries("${CMAKE_SOURCE_DIR}/build/_deps/opensslquic-build/openssl/lib/libssl.a")
-    # nng_link_libraries("${CMAKE_SOURCE_DIR}/build/_deps/opensslquic-build/openssl/lib/libcrypto.a")
+    nng_link_libraries("${CMAKE_SOURCE_DIR}/build/nng/src/supplemental/quic/msquic/msquic/bin/Debug/libmsquic.dylib")
+    nng_link_libraries("${CMAKE_SOURCE_DIR}/build/_deps/opensslquic-build/openssl/lib/libssl.a")
+    nng_link_libraries("${CMAKE_SOURCE_DIR}/build/_deps/opensslquic-build/openssl/lib/libcrypto.a")
```

Comment:
target_link also do the tricks, plz check docs of cmake. Not against this way. but we have obsolete this method

Reply:
I have removed it